### ISSUE TITLE
file browser does not revert on clicking Cancel

### ISSUE THREAD

**Issue by gmarinov** [vizorvr/vizor#1866]
1. vizor.io/edit -> Program -> edit Ball patch
2. click Change on 3D Geometry Loader node
3. click on a filename in the list of files
4. press Cancel

expected: existing model stays
actual: existing model changes and an Undo step is recorded

**Comment by fthr** [comment]
Can confirm. I also thought this was fixed a long time ago, but might have missed. I don't press 'cancel' often.

**Comment by esaruoho** [comment]
Yep, @gmarinov is right, this still happens (1st of july 2017). 

Steps to replicate:

1- Add Texture
2- Click on Browse
3- Select previously uploaded Texture
4- Press cancel

-- Texture-atom starts showing the Texture you just cancelled.
